interfaces/builtin/desktop.go

```diff
@@ -311,6 +311,26 @@ dbus (send)
     member={Check,CheckSub,Get,GetSub,Set,SetSub}
     peer=(label=unconfined),
 
+# Allow for color managed applications to communicate with colord
```

Comment:
Could you add these rules to `desktopConnectedPlugAppArmor` instead? We have colord on Ubuntu Core Desktop, so it'd make sense to grant access there. It is currently running unconfined, so the rules look like they'd be applicable as is.

Reply:
done